packages/clay-modal/src/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {FunctionComponent, HTMLAttributes} from 'react';
+import warning from 'warning';
+
+export interface FooterContainerProps extends HTMLAttributes<HTMLDivElement> {
+	position?: number;
+}
+
+const ButtonWrap: FunctionComponent<HTMLAttributes<HTMLDivElement>> = ({
```

Comment:
Should this exist in `@clayui/button` instead? Or is it only ever used in Modal?

Reply:
I'm not sure if we have use cases outside Modal for the `btn-group-item`, I think it makes sense to keep it there. I'll change that. Maybe we can do the same thing that was done on the radio.